src/status_im/ui/screens/wallet/choose_recipient/views.cljs

```diff
@@ -46,9 +48,10 @@
   (int (* 2 (/ (min height width) 3))))
 
 (defview choose-recipient []
-  (letsubs [read-once?        (atom false)
-            dimensions        [:dimensions/window]
-            camera-flashlight [:wallet.send/camera-flashlight]]
+  (letsubs [{:keys [on-recipient]} [:get-screen-params :recipient-qr-code]
+            camera-flashlight      (reagent/atom :off) ;; values are :off and :on
```

Comment:
how does one switch the camera on here?

Reply:
good catch, missed it

Reply:
added it back, there is a button in the toolbar that toggles the flashlight status